Example/Backpack/SwiftUI/Components/RatingBar/RatingBarExampleView.swift

```diff
@@ -0,0 +1,65 @@
+//
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2023 Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+import Backpack_SwiftUI
+
+struct RatingBarExampleView: View {
+    
+    var body: some View {
+        VStack(spacing: .base) {
+            ratings(title: "Default", .default)
+            ratings(title: "On Contrast", .onContrast)
+                .background(.canvasContrastColor)
+        }
+    }
+    
+    private func ratings(title: String, _ style: BPKRatingBar.Style) -> some View {
+        VStack(spacing: .base) {
+            BPKText(title, style: .heading3)
+            BPKRatingBar(
+                label: "Rating",
+                rating: 2.5,
+                accessibilityLabel: "Rating",
+                style: style
+            )
+            BPKRatingBar(
+                label: "Rating",
+                rating: 2.5,
+                accessibilityLabel: "Rating",
+                ratingScale: .zeroToTen,
+                style: style
+            )
+            BPKRatingBar(
+                label: "Rating",
+                rating: 2.5,
+                accessibilityLabel: "Rating",
+                showScale: false,
+                style: style
+            )
+        }
+        .padding()
```

Comment:
Can we make the accessibility label more descriptive of the actual use. 
If we just do 'rating' we set ourselves up for a11y failures when people fail to understand this will be the only thing that is read out. 

I would even venture and say that we might even miss the leading label as well. 
Should this a11y label only cover the 4.0/5 bit and then we group the label + score as an a11y label?

Reply:
I'd say lets wrap the entire thing, to allow for features to customise what they say. But also the 'out of' part would still be needed to be provided, I believe one prop is easier to deal with this